css/selectors/indeterminate.json

```
@@ -147,7 +149,8 @@
         },
         "radio": {
           "__compat": {
-            "description": "<code>type=&quot;radio&quot;</code>",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/input/radio",
+            "description": "<code>&lt;input type=&quot;radio&quot;&gt;</code>",
```

And I suppose for consistency, we ought to use the longer form on entries like these too

```suggestion
            "description": "Applies to <code>&lt;input type=&quot;radio&quot;&gt;</code>",
```